angular-todo/e2e/register-new-user.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { test, expect } from '@playwright/test';
+import { v4 as uuid } from 'uuid';
+
+const userName = uuid();
+
+test('sign up user angular', async ({ page }) => {
+  await page.goto('https://localhost:8443/home', { waitUntil: 'networkidle' });
```

Comment:
I think we should find an alternative to `networkIdle`

Reply:
set to default and worked(load)

Reply:
I set the 'waitUntil' option to 'load' which is the default option and worked as expected. Since more than one comment with the same question marking it to 'resolved'